common/test/common/InlineStylesTest.scala

```diff
@@ -69,4 +69,10 @@ class InlineStylesTest extends FlatSpec with Matchers {
 
     CSSRule.makeStyles(styles) should be(ListMap("font-family" -> "Arial", "background" -> "red url(data:image/png;base64,ABCDEF)"))
   }
+
+  it should "work with urls in values" in {
+    val styles = "background: url(http://www.example.com/image.png)"
+
+    CSSRule.makeStyles(styles) should be(ListMap("background" -> "url(http://www.example.com/image.png)"))
```

Comment:
i don't know if the css inliner itself should be opinionated about that, obviously the url here is just for this test and in real usage the urls will probably come from something like the `Static` helper in scala - it may be an idea to upgrade that to always use relative, but i think right now all images are served over https which might be better?